src/sentry/runner/commands/run.py

```diff
@@ -484,6 +486,15 @@ def cron(**options: Any) -> None:
 
     from sentry.celery import app
 
+    old_schedule = app.conf.CELERYBEAT_SCHEDULE
+    new_schedule = {}
+    task_schedules = set(options.get("taskworker.scheduler.rollout", []))
+    for key, schedule_data in old_schedule.items():
+        if key not in task_schedules:
+            new_schedule[key] = schedule_data
+
+    app.conf.update(CELERYBEAT_SCHEDULE=new_schedule)
```

Comment:
To confirm my understanding of how this would work:

1. We add the schedules to TASKWORKER_SCHEDULES with the same key/name as they have in CELERYBEAT_SCHEDULES
2. We can use the `scheduler.rollout` option to allow list schedules to taskworkers.
3. The next time schedulers restart their schedules will update.

I think this will work really well for the most part. When we're moving high frequency schedules, we'll have a short period where tasks will be spawned by both celery + taskscheduler as we can't control the restart order of pods.

Reply:
And now that I think it through again, I realize that's not what we want because I want to be able to change the code without having the tasks actually start getting scheduled.